xfail maml test, instead of running it without fake tensor prop

ezyang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #89672
* #89671
* #89670
* #89669
* #89668
* #89666
* #89664
* #89665
* #89662
* #89646
* __->__ #89645
* #89644
* #89643
* #89641
* #89640
* #89639
* #89638
* #89637
* #89631

A previous version of this patch graph breaks when torch.tensor fails, but that causes

```
PYTORCH_TEST_WITH_DYNAMO=1 python test/nn/test_embedding.py -k test_embedding_bag_1D_padding_idx_cpu_float32
```

to start failing. Probably another latent bug that needs investigating.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

cc @mlazos @soumith @voznesenskym @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @chunyuan-w @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

ezyang committed:
Graph break on torch.tensor failure, allowing maml to run with fake tensors

This code is deeply questionable: we should never have fed Tensor
into list items passed to torch.tensor; they should have been SymInt.
But even if they were SymInt this wouldn't work either, since we don't
have SymInt argument support for torch.tensor.  So graph breaking is
best here.

Signed-off-by: Edward Z. Yang <ezyang@fb.com>

[ghstack-poisoned]

ezyang committed:
Update on "Graph break on torch.tensor failure, allowing maml to run with fake tensors"


This code is deeply questionable: we should never have fed Tensor
into list items passed to torch.tensor; they should have been SymInt.
But even if they were SymInt this wouldn't work either, since we don't
have SymInt argument support for torch.tensor.  So graph breaking is
best here.

Signed-off-by: Edward Z. Yang <ezyangfb.com>

cc mlazos soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen chunyuan-w XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx desertfire

[ghstack-poisoned]

albanD reviewed (approved):
Graph break does sound better here.

ezyang commented:
This seems to uncover bugs in graph break handling and tensor in `PYTORCH_TEST_WITH_DYNAMO=1 python test/nn/test_embedding.py -k test_embedding_bag_1D_padding_idx_cpu_float32`. Opting for something less aggressive here.

ezyang committed:
Update on "Graph break on torch.tensor failure, allowing maml to run with fake tensors"


This code is deeply questionable: we should never have fed Tensor
into list items passed to torch.tensor; they should have been SymInt.
But even if they were SymInt this wouldn't work either, since we don't
have SymInt argument support for torch.tensor.  So graph breaking is
best here.

Signed-off-by: Edward Z. Yang <ezyangfb.com>

cc mlazos soumith voznesenskym yanboliang penguinwu anijain2305 EikanWang jgong5 Guobing-Chen chunyuan-w XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx desertfire

[ghstack-poisoned]
